codeconnector.md

```diff
@@ -0,0 +1,42 @@
+## About Code Connector
```

Comment:
This file seems out of place. Should this be under general, or should we remove this, since it is a copy of the website's About Us page?

Reply:
I think the file should be kept, under the assumption that not everyone who stumbles on or is sent to this repository will know of code connector or its site this about blurb will add exposure to exactly what code connector is all about and how to reach out to the code connector community.